tests/test_models/test_inference/test_forecast.py

```diff
@@ -126,6 +111,21 @@ def test_forecast_in_sample_full_no_target_failed_not_enough_context(self, model
         with pytest.raises(ValueError, match="Given context isn't big enough"):
             self._test_forecast_in_sample_full_no_target(example_tsds, model, transforms)
 
+    @to_be_fixed(raises=AssertionError)
+    # Looks like a problem of current implementation of NNs
+    @pytest.mark.parametrize(
+        "model, transforms",
+        [
+            (RNNModel(input_size=1, encoder_length=7, decoder_length=7, trainer_params=dict(max_epochs=1)), []),
+            (
+                MLPModel(input_size=2, hidden_size=[10], decoder_length=7, trainer_params=dict(max_epochs=1)),
+                [LagTransform(in_column="target", lags=[5, 6])],
+            ),
+        ],
+    )
+    def test_forecast_in_sample_full_no_target_failed_assertion_error(self, model, transforms, example_tsds):
```

Comment:
I've seen this pattern while refactoring nn for etna-v2

Why do we duplicate information about assertion raised?

Reply:
Can you explain please what exactly do you mean? Naming of the test?

Reply:
>     @to_be_fixed(raises=AssertionError)
> test_forecast_in_sample_full_no_target_failed_assertion_error

Instead of using test_forecast_in_sample_full_no_target with pytest.mark.xfail

Reply:
* `assertion_error` to distinguish different tests that failed in different ways.
* `to_be_fixed` is used to clarify that we want to fix that test in the future and I can write a particular error message to catch inside the test.